packages/strategy-tests/src/lib.rs

```diff
@@ -1565,7 +1565,11 @@ impl Strategy {
                     DataContract::generate_data_contract_id_v0(partial_identity.id, *identity_nonce);
                 contract.set_id(new_id);
 
-                id_mapping.insert(old_id, new_id); // Store the mapping
+                // Set version to 1
+                contract.set_version(1);
```

Comment:
I recommend to set correct version somewhere above where you are creating start contracts. Contract with version 0 is not valid by definition.

Reply:
Ok. I moved this into the TUI